host-interaction/file-system/copy/copy-file.yml

```diff
@@ -26,8 +26,7 @@ rule:
           - number: 2 = FO_COPY
           - or:
             - api: kernel32.SHFileOperation
-      - call:
-        - and:
-          - number: 2 = FO_COPY
-          - or:
-            - api: kernel32.SHFileOperation
+      - and:
+        - number: 2 = FO_COPY
+        - or:
+          - api: kernel32.SHFileOperation
```

Comment:
this changes the logic for static analysis: the number 2 and API SHFileOperation must be found anywhere within the function, rather than in a basic block (as the prior branch encodes). 

while the `call:` branch here would duplicate the scope declared at the top of the file, i think it still serves a purpose here. so i *think* these lines should be reverted.

Reply:
i think this comment applies to the other files, too.

Reply:
You are right, I will fix this.

Reply:
This should be fixed now